tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -288,11 +292,10 @@ Here is a matrix that is 4 x 3 in size (4 rows and 3 columns):
 
 ### Dimensions in Arrays
 An array contains 1 or more dimensions in the "z" direction. For example, let's 
-say that we collected 
-this same set of species data for every day in a 30 day month. We might then have 
-a matrix like the one above for each day for a total of 30 days making a 4 x 3 x 
-30 array (this dataset has more than 2 dimensions). More on R object types 
-<a href="http://www.statmethods.net/input/datatypes.html">here</a>.
+say that we collected this same set of species data for every day in a 30 day month.
+We might then have a matrix like the one above for each day for a total of 30 days 
+making a 4 x 3 x 30 array (this dataset has more than 2 dimensions). More on R object 
+types <a href="http://www.statmethods.net/input/datatypes.html">here</a>.
```

Comment:
add a bit more to let people know they are headed out of our site "here from DataCamp".

Reply:
Good idea, I added the text "(links to external site, DataCamp)."